SQL/tests/syntax/syntax_test_mysql.sql

```diff
@@ -4919,3 +4951,211 @@ LOAD DATA INFILE 'file.txt'
+--            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ meta.group.sql
+--            ^ punctuation.section.group.begin.sql
+--             ^^^^^^ keyword.other.ddl.sql
+--                    ^^^ keyword.other.ddl.sql
+--                       ^^^^^^^ meta.group.sql
+--                       ^ punctuation.section.group.begin.sql
+--                        ^ meta.number.integer.decimal.sql constant.numeric.value.sql
+--                         ^ punctuation.separator.sequence.sql
+--                          ^ meta.number.integer.decimal.sql constant.numeric.value.sql
+--                           ^ punctuation.separator.sequence.sql
+--                            ^ meta.number.integer.decimal.sql constant.numeric.value.sql
+--                             ^ punctuation.section.group.end.sql
+--                              ^ punctuation.separator.sequence.sql
+--                               ^^^ keyword.other.ddl.sql
+--                                  ^^^^^^^ meta.group.sql
+--                                  ^ punctuation.section.group.begin.sql
+--                                   ^ meta.number.integer.decimal.sql constant.numeric.value.sql
+--                                    ^ punctuation.separator.sequence.sql
+--                                     ^ meta.number.integer.decimal.sql constant.numeric.value.sql
+--                                      ^ punctuation.separator.sequence.sql
+--                                       ^ meta.number.integer.decimal.sql constant.numeric.value.sql
+--                                        ^ punctuation.section.group.end.sql
+--                                         ^ punctuation.separator.sequence.sql
+--                                          ^^^ keyword.other.ddl.sql
+--                                             ^^^^^^^ meta.group.sql
+--                                             ^ punctuation.section.group.begin.sql
+--                                              ^ meta.number.integer.decimal.sql constant.numeric.value.sql
+--                                               ^ punctuation.separator.sequence.sql
+--                                                ^ meta.number.integer.decimal.sql constant.numeric.value.sql
+--                                                 ^ punctuation.separator.sequence.sql
+--                                                  ^ meta.number.integer.decimal.sql constant.numeric.value.sql
+--                                                   ^ punctuation.section.group.end.sql
+--                                                    ^ punctuation.section.group.end.sql
+--                                                      ^^ keyword.operator.assignment.alias.sql
+--                                                         ^ meta.alias.table.sql
+    INTO OUTFILE '/tmp/select-values.txt';
+
+TABLE employees ORDER BY lname LIMIT 1000
+-- ^^ keyword.context.mysql
+--    ^^^^^^^^^ meta.table-name.sql
+-- ^^^ - meta.table-name
```

Comment:
Just for the sake of perfection ;)

```suggestion
-- ^^^ - meta.table-name
--    ^^^^^^^^^ meta.table-name.sql
```